src/lib.rs

```diff
@@ -15,8 +73,23 @@ use std::fs::{self, File, OpenOptions};
 use std::io::{Error as IoError, ErrorKind::NotFound, Write};
 use std::path::PathBuf;
 
-/// Load a configuration from the standard OS local scope for
-/// the current user.
+/// Load an application configuration from disk
+///
+/// A new configuration file is created with default values if none
+/// exists.
+///
+/// Errors that are returned from this function are IO related,
+/// for example if the writing of the new configuration fails
+/// or `confy` encounters an operating system or environment
+/// that it does not support.
+///
+/// **Note:** The type of configuration needs to be declared in some way
+/// that is inferrable by the compiler. Also note that your
+/// configuration needs to implement `Default`.
+///
+/// ```rust,ignore
+///     let cfg: MyConfig = confy::load("my-app")?;
```

Comment:
identation

can you make this `no_run`?

Reply:
`MyConfig` wouldn't actually exist in this scope though